src/webgl/shaders/imageLightDiffused.frag

```diff
@@ -0,0 +1,82 @@
+precision mediump float;
```

Comment:
oh actually we probably want this to be `highp`; this basically gets ignored by the webgl driver on desktop, but on mobile it's actually used, and in my experience any time we have normals, the precision makes a big difference on the visual fidelity
```suggestion
precision highp float;
```